controller/node_controller.go

```diff
@@ -1027,60 +1031,66 @@ func (nc *NodeController) syncMultipathd(node *longhorn.Node, namespaces []lhtyp
 	node.Status.Conditions = types.SetCondition(node.Status.Conditions, longhorn.NodeConditionTypeMultipathd, longhorn.ConditionStatusTrue, "", "")
 }
 
-func (nc *NodeController) syncNFSClientVersion(kubeNode *corev1.Node, node *longhorn.Node, namespaces []lhtypes.Namespace) {
-	kernelVersion := kubeNode.Status.NodeInfo.KernelVersion
-	nfsClientVersions := []string{"CONFIG_NFS_V4_2", "CONFIG_NFS_V4_1", "CONFIG_NFS_V4"}
+func (nc *NodeController) checkModulesLoaded(kubeNode *corev1.Node, node *longhorn.Node, namespaces []lhtypes.Namespace) {
+	modules := map[string]string{
+		"CONFIG_DM_CRYPT": "dm_crypt",
+	}
```

Comment:
NIT: If it's better to make this const or a separate get func

Reply:
Modified. Moved to the top as `var()` with `nfsClientVersions`. Thanks.